src/CMakeLists.txt

```diff
@@ -357,6 +357,8 @@ if (NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
   add_subdirectory(par)
 else()
   message(STATUS "Removing MPL2 and PAR to avoid run time fatal error.")
+  target_compile_definitions(openroad PRIVATE ENABLE_MPL2=0)
+  target_compile_definitions(openroad PRIVATE ENABLE_PAR=0)
```

Comment:
I don't think this helps. The code checks whether it's defined, not what the value is.